work-scheduler-data/src/main/java/com/github/gaalgergo/workscheduler/data/repositories/EmployeeRepository.java

```diff
@@ -5,4 +5,8 @@
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface EmployeeRepository extends JpaRepository<EmployeeEntity, Long> { }
+public interface EmployeeRepository extends JpaRepository<EmployeeEntity, Long> {
+
+    @Override
+    <S extends EmployeeEntity> S save(S s);
```

Comment:
You don't need to override this interface method. It is already part of the `JpaRepository` interface.
Remove this method declaration.